### PR TITLE
GRW-2280 / Date-input-design-updates

### PR DESCRIPTION
**Pull request by amandaflod**
## Describe your changes

Update the date input field to match new designs. 

https://user-images.githubusercontent.com/59652803/220919606-04179d3b-f792-440a-a3d5-a0516c2986cb.mov

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-2280]: https://hedvig.atlassian.net/browse/GRW-2280?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx [hunk 1]
```diff
@@ -134,17 +134,14 @@ type BankSigneringInvalidRenewalDateProps = Pick<Props, 'onStartDateChange' | 's
 const BankSigneringInvalidRenewalDateCancellation = (
   props: BankSigneringInvalidRenewalDateProps,
 ) => {
-  const { onStartDateChange, companyName, startDate } = props
-  const { t } = useTranslation('purchase-form')
-  const message = t('AUTO_SWITCH_INVALID_RENEWAL_DATE_MESSAGE', { company: companyName })
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx:139]: are we not using this translation anymore? did you remove from Lokalise?

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/DateInput.tsx [hunk 2]
```diff
@@ -33,107 +26,104 @@ export const DateInput = (props: Props) => {
...
   position: 'relative',
   display: 'flex',
-  gap: theme.space.xxs,
-  justifyContent: 'space-between',
-  alignItems: 'center',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  borderRadius: theme.radius.sm,
+  backgroundColor: theme.colors.gray100,
+  height: '3.25rem',
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/DateInput.tsx:57]: I discussed with Zak -- these should be 48px :)

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/DateInput.tsx [hunk 3]
```diff
@@ -33,107 +26,104 @@ export const DateInput = (props: Props) => {
...
-
-  const handleBlur: FocusEventHandler<HTMLInputElement> = (event) => {
-    setFocused(false)
-    onBlur?.(event)
-  }
-
-  const floatingLabel = dateValue ? (
-    <Text>{formatter.fromNow(dateValue)}</Text>
-  ) : (
-    <Text color="textSecondary">{t('DATE_INPUT_EMPTY_LABEL')}</Text>
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/DateInput.tsx:49]: same question: did we remove the translation from lokalise or are we gonna use it?

**Review by robinandeer — COMMENTED** [review]
Neat stuff! Just had some follow up comments

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx (hunk 1 above)
Comment (amandaflod) [reply apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx:139]: I was unable to remove this one in lokalise, will have to try again later!

**Review by robinandeer — APPROVED** [review]

**Pull request merged**